src/tests/artifacts/.shepherd/prefs.yaml

```diff
@@ -0,0 +1 @@
+workingDirectory: /Users/aorinevo/Repositories/nerdwallet/shepherd/src/tests/artifacts/.shepherd
```

Comment:
Nit: this path looks specific to your machine - consider adding this file to `.gitignore`.

Reply:
Will do.

Reply:
I think I'll just remove it.